pkg/reconciler/githubsource/githubsource.go

```diff
@@ -58,17 +61,23 @@ func Add(mgr manager.Manager, logger *zap.SugaredLogger) error {
 	if !defined {
 		return fmt.Errorf("required environment variable %q not defined", raImageEnvVar)
 	}
+	scheme := mgr.GetScheme()
+	r := &reconciler{
+		recorder:            mgr.GetRecorder(controllerAgentName),
+		scheme:              scheme,
+		receiveAdapterImage: receiveAdapterImage,
+		webhookClient:       gitHubWebhookClient{},
+		eventTypeReconciler: eventtype.Reconciler{
+			Scheme: scheme,
+		},
+	}
 
+	log.Println("Adding the GitHub Source controller.")
 	p := &sdk.Provider{
-		AgentName: controllerAgentName,
```

Comment:
I would fold this all back up again and just call mgr.GetScheme() twice.

Reply:
Done, rollbacked for all four source controllers that I changed.